apps/sap-commerce-cloud/src/components/Dialog/Dialog.styles.ts

```diff
@@ -0,0 +1,12 @@
+import { css } from 'emotion';
+import tokens from '@contentful/forma-36-tokens';
+
+export const styles = {
+  textInput: css({ width: '250px' }),
+  grid: css({ marginTop: tokens.spacingM, paddingLeft: tokens.spacingL }),
+  table: css({ padding: tokens.spacingL }),
+  tableCell: css({ width: '10%' }),
+  pagination: css({ margin: tokens.spacingL }),
+  nextButton: (page: number) => css({ marginLeft: page > 0 ? tokens.spacingL : '' }),
```

Comment:
I noticed with this next page pagination button, the left margin is always present currently. We only want that spacing when there's also a previous page button so I added in this new logic to handle.